bundles/org.openhab.binding.opensprinkler/src/main/resources/ESH-INF/binding/binding.xml

```diff
@@ -6,6 +6,6 @@
 
 	<name>OpenSprinkler Binding</name>
 	<description>This is the binding for OpenSprinkler.</description>
-	<author>Chris Graham</author>
+	<author>Chris Graham and Florian Schmidt</author>
```

Comment:
Usually we don't update that and leave the original author in.

Reply:
The original author is still there? :/

Reply:
I should rephrase that: Usually we leave the original author and don't touch that. Better add your contribution in the source files as`@author` with a description of your changes. And if you made larger contributions (like this one), add yourself as a CODEOWNER. You will then be proposed as reviewer if someone else adds changes (which makes sense since it you have a good overview of the current code),

Reply:
I generally dislike the idea of holding changes in the file as a header, as this somehow duplicates the git history, is error prone (as no mandatory step) and blows up the source code. But hey, if it's done in that way here, I'll do it.